app/views/forms/_screen.html.erb

```diff
@@ -37,8 +37,101 @@
 
     </div>
 
-    <%# questions on the form %>
-    <%= index_table(Questioning, @form.questionings, :within_form => true) if form_mode != :new %>
+    <div class="questions">
+      <input class="btn btn-primary" type="submit" value="Save"></input>
+
+      <h2>Questions</h2>
+
+      <div class="questions-header" id="questions-header">
```

Comment:
Check out the top-action-links style on main.css.scss line 268. I think you could use that here.